packages/frameworks-sveltekit/package.json

```diff
@@ -48,6 +48,8 @@
     "@auth/core": "workspace:*"
   },
   "peerDependencies": {
+    "cookie": "0.6.0",
+    "@types/cookie": "0.6.0",
```

Comment:
should not be a peer dependency, we don't want the user to need to install this.

Reply:
Yeah these are deps of `@auth/core` already so i wasn't sure where to put them here exactly. Since folks are almost always installed the two together, know what i mean